pkg/loadbalancerservice/controller.go

```diff
@@ -0,0 +1,154 @@
+package loadbalancerservice
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/cloud-provider/service/helpers"
+	"k8s.io/klog/v2"
+
+	"github.com/openshift/microshift/pkg/config"
+)
+
+const defaultInformerResyncPeriod = 10 * time.Minute
+
+type LoadbalancerServiceController struct {
+	sync.Mutex
+	NodeName   string
+	NodeIP     string
+	KubeConfig string
+	client     *kubernetes.Clientset
+	myIPs      []string
```

Comment:
I don't see `myIPs` or `hostCount` used in the loadbalancer controller. why do we need them?

Reply:
removed